demos/include/iot_demo_runner.h

```diff
@@ -174,6 +174,8 @@
     #define DEMO_entryFUNCTION             RunHttpsAsyncUploadDemo
 #elif defined( CONFIG_MQTT_BLE_TRANSPORT_DEMO_ENABLED )
```

Comment:
If the config is named as `CONFIG_MQTT_BLE_TRANSPORT_DEMO_ENABLED`, then the name of the metadata ID should be `MQTT_BLE_TRANSPORT_DEMO` (as the FreeRTOS console service adds the `CONFIG` and `ENABLED`) prefix when generating the Project to download.

Similarly, for the `CONFIG_SHADOW_BLE_TRANSPORT_DEMO_ENABLED` macro. Make sure to update the metadata ID for the Shadow demo to be  `SHADOW_BLE_TRANSPORT_DEMO`